File: 백준/Bronze/11549. Identifying tea/Identifying tea.java
import java.util.*;
public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int arr[] = new int [5];
        int cnt = 0;
        for(int i = 0; i < 5; i++){
            arr[i] = sc.nextInt();
            if(arr[i] == n)
                cnt++;
        }
        System.out.println(cnt);
    }
}

File: 백준/Bronze/11549. Identifying tea/Identifying tea.c
#include<stdio.h>
int main(void){
    int n;
    int cnt = 0;
    scanf("%d",&n);
    for(int i = 0; i < 5; i++){
        int T;
        scanf("%d",&T);
        if(n == T)
            cnt++;
    }
    printf("%d\n",cnt);
    return 0;
}